### ISSUE TITLE
Incorporar inscripción a clases existentes desde la web de los alumnos

### ISSUE THREAD

**Issue by manuelbicain** [bcardiff/themis#71]
Esto es un add GIGANTE que creo podemos descomponer en otras cosas previas. Implica, creo, hacer más fuerte el logueo a la app, redimensionar el server tal vez, hacer alguna consulta de a qué están inscriptos, etc. 
Pero lo dejo acá para arrancar en algún momento.

**Comment by manuelbicain** [comment]
Agrego cosas que se desprenden:

Mail al alumno de confirmación. 
Alerta de clases llegando a un maximo
Alerta de clases sin llegar a un mínimo para cerrarlas 
Cada alumno con un usuario de ser posible login con facebook o goolge, bla, bla.
Que el alumno avise si toma la clase online. 
Que el sistema le avise al profesor por ejemplo en el pad de “presentes” que hay gente online.

**Comment by Mar12358** [comment]
Sí, es gigante, pero es lo que todos queremos. Queda para mas adelante.
